rotkehlchen/premium/premium.py

```diff
@@ -228,11 +234,15 @@ def upload_data(
             'API-SIGN': base64.b64encode(signature.digest()),
         })
 
+        tmp_file = io.BytesIO()
+        tmp_file.write(data_blob)
+        tmp_file.seek(0)
         try:
-            response = self.session.put(
-                self.uri + 'save_data',
+            response = self.session.post(
+                self.rotki_nest + '1/backup',
```

Comment:
If you do what I said to do in `__init__`

```suggestion
                self.rotki_nest + '/backup',
```